converter.go

```diff
@@ -1,158 +1,169 @@
+	// Read body of API response
+	body, err := ioutil.ReadAll(res.Body)
+	res.Body.Close()
+	check(err)
 
-    // Unmarshal json into Features struct
-    features := geojson.NewFeatureCollection()
-    err = json.Unmarshal(body, features)
-    check(err)
+	// Unmarshal json into Features struct
+	features := geojson.NewFeatureCollection()
+	err = json.Unmarshal(body, features)
+	check(err)
 
-    convertedFeatures := geojson.NewFeatureCollection()
-    err = json.Unmarshal(body, convertedFeatures)
-    check(err)
+	convertedFeatures := geojson.NewFeatureCollection()
+	err = json.Unmarshal(body, convertedFeatures)
+	check(err)
 
-    return features, convertedFeatures
+	return features, convertedFeatures
 }
 
 /*
 Simple error checking function.
-*/    
+*/
 func check(e error) {
-    if e != nil {
-        panic(e)
-    }
+	if e != nil {
+		panic(e)
+	}
 }
 
 /*
 Converts a set of coordinates using cs2cs. http://proj4.org/apps/cs2cs.html
 */
 func convertCoordinates(lon, lat float64) (float64, float64) {
-    // The coordinates are floats, so we must convert them into strings to include them in the cs2cs command
-    lonStr := strconv.FormatFloat(lon, 'f', -1, 64)
-    latStr := strconv.FormatFloat(lat, 'f', -1, 64)
-    
-    var cs2csProjection string
-
-    // Check whether coordinates are NAD83 or Web Mercator. NAD83 will have a positive longitude given our region of the globe.
-    // TODO: ARCGIS will soon only be using NAD83, so this if/else statement won't be necessary.
-    if lon > 0 {
-        cs2csProjection = "+proj=lcc +lat_1=46 +lat_2=44.33333333333334 +lat_0=43.66666666666666 +lon_0=-120.5 +x_0=2500000.0001424 +y_0=0 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=ft +no_defs"
-    } else {
-        cs2csProjection = "+proj=merc +a=6378137 +b=6378137 +lat_ts=0.0 +lon_0=0.0 +x_0=0.0 +y_0=0 +k=1.0 +units=m +nadgrids=@null +wktext  +no_defs"
-    }
-
-    cs2csCommand := "echo \"" + lonStr + " " + latStr + "\" | cs2cs -f %8.6f " + cs2csProjection + " +to +proj=longlat +datum=WGS84 +no_defs"
-    cmd := exec.Command("sh", "-c", cs2csCommand)
-    stdoutStderr, err := cmd.CombinedOutput()
-    check(err)
-
-    // Cs2cs outputs coordinates separated by random amounts of whitespace, with a 0 value at the end.
-    // strings.Fields parses the 3 numbers so we can use them individually (and ignore the unneeded 0).
-    parsedOutput := strings.Fields(string(stdoutStderr[:]))
-
-    convertedLon, err := strconv.ParseFloat(parsedOutput[0], 64)
-    check(err)
-    convertedLat, err := strconv.ParseFloat(parsedOutput[1], 64)
-    check(err)
-
-    return convertedLon, convertedLat
+	// The coordinates are floats, so we must convert them into strings to include them in the cs2cs command
+	lonStr := strconv.FormatFloat(lon, 'f', -1, 64)
+	latStr := strconv.FormatFloat(lat, 'f', -1, 64)
+
+	var cs2csProjection string
+
+	// Check whether coordinates are NAD83 or Web Mercator. NAD83 will have a positive longitude given our region of the globe.
```

Comment:
Let's remove these comments and commented code since it was replaced with line 70.

Reply:
Fixed